docs/gmi.md

```diff
@@ -47,11 +47,14 @@ e.g. console.log.
 
 ## Stats
 
-The sending of stats is now handled by GMI. A simple call can be used:
+The sending of stats is handled by GMI. 2 calls can are needed:
```

Comment:
Think this needs a bit of rewording: "2 calls can are needed"

Reply:
Yeah that made no sense :) Fixed now.